src/afl-fuzz-redqueen.c

```diff
@@ -167,6 +167,17 @@ static u8 get_exec_checksum(afl_state_t *afl, u8 *buf, u32 len, u64 *cksum) {
 
 }
 
+/* replace everything with different values */
+static void random_replace(afl_state_t *afl, u8 *buf, u32 len) {
+
+  for (u32 i = 0; i < len; i++) {
+
+    buf[i] = rand_below(afl, 256);
```

Comment:
would it not be better to do something like:
```
for (...) {
  u8 c;
  do { c = rand_below(afl, 256);  } while (c == buf[i]);
  buf[i] = c;
}
```

Reply:
I think it depends on how successful replacement is going to be. If many bytes are identical and cannot be colorized, forcing those that were colorizable to be changed would be beneficial. If every byte is colorizable, then there's slightly more entropy in allowing 256 possible values rather than 255.

I suspect you're right in the majority of cases, I'll add the changes.